www/common/common-ui-elements.js

```diff
@@ -638,34 +644,28 @@ define([
                 }
                 break;
             case 'import':
-                button = $('<button>', {
-                    'class': 'fa fa-upload cp-toolbar-icon-import',
-                    title: Messages.importButtonTitle,
-                }).append($('<span>', {'class': 'cp-toolbar-drawer-element'}).text(Messages.importButton));
-                    var importer = importContent((data && data.binary) ? 'application/octet-stream' : 'text/plain', callback, {
-                        accept: data ? data.accept : undefined,
-                        binary: data ? data.binary : undefined
-                    });
-
-                    var handler = data.first? function () {
-                        data.first(function () {
-                            importer(); // Make sure we don't pass arguments to importer
-                        });
-                    }: importer; //importContent;
+                button = createIconButton('fa fa-upload cp-toolbar-icon-import', '', Messages.importButtonTitle, Messages.importButton);
```

Comment:
In doubt assign only the icon classes to the <i> element (`fa`, `fa-icon`, `cptools`, `cpools-icon`) and leave the other classes such as `cp-toolbar-icon-print` on the <a> container